dev-packages/browser-integration-tests/suites/integrations/captureConsole-attachStackTrace/init.js

```diff
@@ -6,6 +6,5 @@ window.Sentry = Sentry;
 Sentry.init({
   dsn: 'https://public@dsn.ingest.sentry.io/1337',
   integrations: [captureConsoleIntegration()],
-  autoSessionTracking: false,
```

Comment:
iirc, we opted out of session tracking for some tests due to session envelopes colliding with the envelope we're waiting for. Do we need to filter on `browserSessionIntegration` instead here? Might as well be old code and this is no longer a concern. I guess we'd find out with the flaky test detector 🤔

Reply:
Good point actually. I didn't consider that. I agree though that we would probably catch anything weird in the flakey test detector. Maybe the `autoSessionTracking`s here are also from copy and pasting tests.

I will merge this as is cause I'd err on removing unneeded logic. If there are any CI failures coming up we may attribute it to this.